packages/docs/pages/wallet/web-wallet.mdx

```diff
@@ -1,24 +1,31 @@
 
-### Installing from source (dev and experiment purposes)
+### Installing from source (for development and experiment purposes)
 
 #### Connect to a testnet or run a local node
-1. Follow the instructions in the [testnets](../introduction/testnets/intro.md) to connect to a testnet or you can set up a local node using [docker](../introduction/install/from-docker.md).
-2. Figure out where the base directory is stored and save it as a variable such as `export BASE_DIR=<path/to/base/dir>`. You can follow [these docs](../introduction/testnets/migrating-testnets.md#after-v0153) to save this variable. Go ahead and save the chain id as a variable as well. You can find the chain id by running `cat $BASE_DIR/global-config.toml`. Save this chain-id to the variable `export CHAIN_ID=<CHAIN_ID>`.
-3. You will need to edit the cometbft config in order to allow the web wallet to connect to your node. The cometbft config will be located in `$BASE_DIR/$CHAIN_ID/cometbft/config/config.toml`. You will need to change the `cors_allowed_origins` field to `["*"]`. You can do this by running `sed -i 's/cors_allowed_origins = \[\]/cors_allowed_origins = ["*"]/' $BASE_DIR/$CHAIN_ID/cometbft/config/config.toml`.
+1. Follow the instructions for the [testnets](../introduction/testnets.md) to connect to a testnet or set up a local node using [docker](../introduction/install/docker.md).
+2. Figure out where the base directory is stored and save its location as a variable such as `export BASE_DIR=<path/to/base/dir>`.
+   You can follow [these docs](../introduction/testnets/migrating-testnets.md#after-v0153) to save this variable. Go ahead and save the chain id as a variable as well. You can find the chain id by running `cat $BASE_DIR/global-config.toml`. Save this chain-id to the variable `export CHAIN_ID=<CHAIN_ID>`.
+3. You will need to edit the CometBFT config in order to allow the web wallet to connect to your node.
+   The CometBFT config will be located in `$BASE_DIR/$CHAIN_ID/cometbft/config/config.toml`. You will need to change the `cors_allowed_origins` field to `["*"]`. You can do this by running
+   ```shell copy
+   sed -i 's/cors_allowed_origins = \[\]/cors_allowed_origins = ["*"]/' \
+           $BASE_DIR/$CHAIN_ID/cometbft/config/config.toml
+   ```
 
 #### Setting up the extension
-1. Clone the [namada-interface repository](https://github.com/anoma/namada-interface)
-2. Follow the installation instructions in the intro.md (which should be visible on the repository page).
+1. Clone the [namada-interface repository](https://github.com/anoma/namada-interface).
+2. Follow the installation instructions in `README.md`.
 3. `cd` into the `namada-interface/apps/extension` directory and run `yarn start:chrome`. This will build the extension and place it in the `namada-interface/apps/extension/build` directory. It also starts the dev server which will watch for changes.
```

Comment:
Are these instructions up-to-date? Should we instead ask to follow the instructions from README.md?

Reply:
Since I last looked, yes. Not sure as of current.